### ISSUE TITLE
RN 0.57.2: peer dep missing: @babel/core@7.0.0-beta.38, required by @babel/plugin-check-constants@7.0.0-beta.38

### ISSUE THREAD

**Issue by Strate** [facebook/react-native#21592]
- [x] Review the documentation: https://facebook.github.io/react-native
  - [x] Search for existing issues: https://github.com/facebook/react-native/issues
  - [x] Use the latest React Native release: https://github.com/facebook/react-native/releases

## Environment
```
% npx react-native info    
  React Native Environment Info:
    System:
      OS: Linux 4.15 Ubuntu 16.04.5 LTS (Xenial Xerus)
      CPU: x64 Intel(R) Core(TM) i5-8600 CPU @ 3.10GHz
      Memory: 17.27 GB / 31.35 GB
      Shell: 5.1.1 - /usr/bin/zsh
    Binaries:
      Node: 10.11.0 - /usr/bin/node
      Yarn: 1.10.0 - /usr/local/bin/yarn
      npm: 6.4.1 - /usr/bin/npm
    SDKs:
      Android SDK:
        Build Tools: 23.0.1, 26.0.1, 26.0.3, 27.0.3, 28.0.1, 28.0.2
        API Levels: 23, 26, 27, 28
    npmPackages:
      @types/react: ^16.4.16 => 16.4.16 
      @types/react-native: 0.57.4 => 0.57.4 
      react: 16.5.0 => 16.5.0 
      react-native: 0.57.2 => 0.57.2 
    npmGlobalPackages:
      create-react-native-app: 1.0.0
      react-native-cli: 2.0.1
      react-native-git-upgrade: 0.2.7
```

## Description
There is a conflict in peer dependencies of babel-preset-fbjs and dependencies of react-native itself:
```
npm ERR! peer dep missing: @babel/core@7.0.0-beta.38, required by @babel/plugin-check-constants@7.0.0-beta.38

% npm ls @babel/plugin-check-constants                             
@chatium/mobile@0.0.1 /devel/src/chatium/gbn/mobile
└─┬ react-native@0.57.2
  └─┬ metro@0.47.1
    └─┬ babel-preset-fbjs@3.0.1
      └── @babel/plugin-check-constants@7.0.0-beta.38 

% npm ls @babel/core                  
@chatium/mobile@0.0.1 /devel/src/chatium/gbn/mobile
└─┬ react-native@0.57.2
  ├─┬ metro@0.47.1
  │ └── @babel/core@7.1.2 
  └─┬ metro-babel-register@0.47.1
    └── @babel/core@7.1.2  deduped
```
Upgrading metro to 0.48 should do the trick, as there is removed `babel-preset-fbjs`

**Comment by kelset** [comment]
Thanks for reporting, I never saw this error so I wonder why it's showing for you - btw unless you have a fixed dep to Metro 0.47.x if you clean up your modules and reinstall you should get Metro 0.48 now (since it's out) 🤔

But anyway we are planning to do a new release soon with 0.48

**Comment by Strate** [comment]
@kelset warning happens ever on fresh install, because `react-native@0.57.2` has a dependency of `metro` defined as `^0.47.1`, which is definetely do not cover `0.48` :)

**Comment by kelset** [comment]
Yes sorry, I was thinking about `react-native init` (which should pull latest Metro).

Just to make sure I understand, what the console shows you is an Error or a Warning?
Because if it's just a warning I think we can close this, I mean it's "already" fixed via Metro 0.48 so if you need to enforce it you can use yarn `resolutions` to make sure you are using it... no? 🤔
